_includes/footer-scripts.html

```diff
@@ -33,5 +33,4 @@
   {% endfor %}
 {% endif %}
 
-{% include google_analytics.html %}
-{% include search-init.html %}
\ No newline at end of file
+{% include google_analytics.html %}
```

Comment:
Can you remove that altogether?

Reply:
It is disabled at config level already:
https://github.com/ethereum/solidity-blog/blob/master/_config.yml#L104
So there shouldn't be any mention to Google services in the generated code.

I see a risk of changing this repository quite a lot, making harder to merge important changes from the original blog. As a reminder, I made substantial changes to it recently and might do some more in the future.

For cases like this, we can track in this issue and it won't be forgot.
https://github.com/ethereum/blog/issues/359